src/more-infos/more-info-media_player.html

```diff
@@ -106,6 +106,18 @@
           </paper-listbox>
         </paper-dropdown-menu>
       </div>
+      <!-- SOUND MODE PICKER -->
+      <div class='controls layout horizontal justified'
+           hidden$='[[computeHideSelectSoundMode(isOff, supportsSelectSoundMode)]]'>
```

Comment:
dom-if it instead

Reply:
I don't know exactly what you mean but I guess you mean to use a "dom-if" property in the <div> instead of using the hidden$.
Again I am not an expert in how this works and what the diffrence is.
But the rest of the file is using the hidden$ property.

Basically I made a copy of the Source input, so everything is done in a similar way as the already implemented Source input.

Would it not be better to keep everything the same as the rest of the file for now.
Maybe someone else who does no what the diffrence is and nows how to use these properties correctly can change it for the whole file at once (also for the SourceInput, Power and Volume).